spyder_kernels/customize/spydercustomize.py

```diff
@@ -30,7 +31,8 @@
 from spyder_kernels.customize.umr import UserModuleReloader
 
 if not PY2:
```

Comment:
The `inputtransformer2` module was introduced in IPython 7, so this condition needs to be changed in the same way as I did in https://github.com/spyder-ide/spyder/pull/13721

Reply:
Probably the `if`s you modified below need to be adjusted in the same way, but I'm not really sure.

Reply:
should be fixed